.github/workflows/master.yml

```diff
@@ -21,8 +21,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - run: sudo apt-get update
-      if: matrix.target == 'x86_64-unknown-linux-musl'
```

Comment:
You should not remove this condition: only in musl case we need to update apt packages